### ISSUE TITLE
Wrong type hint

### ISSUE THREAD

**Issue by jmontoyaa** [sonata-project/SonataFormatterBundle#370]
I have this error:

Argument 2 passed to Sonata\FormatterBundle\Controller\CkeditorAdminController::setFormTheme() must be of the type string, array given, called in vendor/sonata-project/formatter-bundle/src/Controller/CkeditorAdminController.php on line 78

#### Sonata packages

```
composer show --latest 'sonata-project/*'
sonata-project/admin-bundle              3.40.1          3.40.1             The missing Symfony Admin Generator
sonata-project/block-bundle              3.12.1          3.12.1             Symfony SonataBlockBundle
sonata-project/cache                     1.1.1           2.0.1              Cache library
sonata-project/cache-bundle              2.4.2           3.0.0              This bundle provides caching services
sonata-project/classification-bundle     3.7.1           3.7.1              Symfony SonataClassificationBundle
sonata-project/core-bundle               3.11.2          3.11.2             Symfony SonataCoreBundle
sonata-project/datagrid-bundle           2.3.1           2.3.1              Symfony SonataDatagridBundle
sonata-project/doctrine-extensions       1.x-dev de0c00b dev-master 3772283 Doctrine2 behavioral extensions
sonata-project/doctrine-orm-admin-bundle 3.6.2           3.6.2              Symfony Sonata / Integrate Doctrine ORM into the SonataAdminBundle
sonata-project/easy-extends-bundle       2.5.0           2.5.0              Symfony SonataEasyExtendsBundle
sonata-project/exporter                  1.9.1           1.9.1              Lightweight Exporter library
sonata-project/formatter-bundle          4.1.1           4.1.1              Symfony SonataFormatterBundle
sonata-project/intl-bundle               2.5.0           2.5.0              Symfony SonataIntlBundle
sonata-project/media-bundle              3.x-dev 9628861 dev-master 5482a9a Symfony SonataMediaBundle
sonata-project/notification-bundle       3.5.1           3.5.1              Symfony SonataNotificationBundle
sonata-project/page-bundle               3.9.0           3.9.0              This bundle provides a Site and Page management through container an...
sonata-project/seo-bundle                2.6.0           2.6.0              Symfony SonataSeoBundle
sonata-project/timeline-bundle           3.3.1           3.3.1              Integrates SpyTimelineBundle into Sonata
sonata-project/user-bundle               4.2.3           4.2.3              Symfony SonataUserBundle

```
#### Symfony packages

```
 composer show --latest 'symfony/*'
symfony/annotations-pack     v1.0.0 Package symfony/annotations-pack is abandoned, you should avoid using it. No replacement was suggested.
 v1.0.0  A pack for the Doctrine annotations with cache
symfony/apache-pack          v1.0.1  v1.0.1  A pack for Apache support in Symfony
symfony/asset                v4.1.6  v4.1.6  Symfony Asset Component
symfony/browser-kit          v4.1.6  v4.1.6  Symfony BrowserKit Component
symfony/cache                v4.1.6  v4.1.6  Symfony Cache component with PSR-6, PSR-16, and tags
symfony/class-loader         v3.4.17 v3.4.17 Symfony ClassLoader Component
symfony/config               v4.1.6  v4.1.6  Symfony Config Component
symfony/console              v4.1.6  v4.1.6  Symfony Console Component
symfony/css-selector         v4.1.6  v4.1.6  Symfony CssSelector Component
symfony/debug                v4.1.6  v4.1.6  Symfony Debug Component
symfony/debug-bundle         v4.1.6  v4.1.6  Symfony DebugBundle
symfony/dependency-injection v4.1.6  v4.1.6  Symfony DependencyInjection Component
symfony/doctrine-bridge      v4.1.6  v4.1.6  Symfony Doctrine Bridge
symfony/dom-crawler          v4.1.6  v4.1.6  Symfony DomCrawler Component
symfony/dotenv               v4.1.6  v4.1.6  Registers environment variables from a .env file
symfony/event-dispatcher     v4.1.6  v4.1.6  Symfony EventDispatcher Component
symfony/expression-language  v4.1.6  v4.1.6  Symfony ExpressionLanguage Component
symfony/filesystem           v4.1.6  v4.1.6  Symfony Filesystem Component
symfony/finder               v4.1.6  v4.1.6  Symfony Finder Component
symfony/flex                 v1.1.1  v1.1.1  Composer plugin for Symfony
symfony/form                 v4.1.6  v4.1.6  Symfony Form Component
symfony/framework-bundle     v4.1.6  v4.1.6  Symfony FrameworkBundle
symfony/http-foundation      v4.1.6  v4.1.6  Symfony HttpFoundation Component
symfony/http-kernel          v4.1.6  v4.1.6  Symfony HttpKernel Component
symfony/inflector            v4.1.6  v4.1.6  Symfony Inflector Component
symfony/intl                 v4.1.6  v4.1.6  A PHP replacement layer for the C intl extension that includes additional data from the ICU library.
symfony/monolog-bridge       v4.1.6  v4.1.6  Symfony Monolog Bridge
symfony/monolog-bundle       v3.3.0  v3.3.0  Symfony MonologBundle
symfony/options-resolver     v4.1.6  v4.1.6  Symfony OptionsResolver Component
symfony/phpunit-bridge       v4.1.6  v4.1.6  Symfony PHPUnit Bridge
symfony/polyfill-apcu        v1.9.0  v1.9.0  Symfony polyfill backporting apcu_* functions to lower PHP versions
symfony/polyfill-ctype       v1.9.0  v1.9.0  Symfony polyfill for ctype functions
symfony/polyfill-intl-icu    v1.9.0  v1.9.0  Symfony polyfill for intl's ICU-related data and classes
symfony/polyfill-mbstring    v1.9.0  v1.9.0  Symfony polyfill for the Mbstring extension
symfony/polyfill-php70       v1.9.0  v1.9.0  Symfony polyfill backporting some PHP 7.0+ features to lower PHP versions
symfony/polyfill-php72       v1.9.0  v1.9.0  Symfony polyfill backporting some PHP 7.2+ features to lower PHP versions
symfony/process              v3.4.17 v4.1.6  Symfony Process Component
symfony/property-access      v4.1.6  v4.1.6  Symfony PropertyAccess Component
symfony/requirements-checker v1.1.2  v1.1.2  Check Symfony requirements and give recommendations
symfony/routing              v4.1.6  v4.1.6  Symfony Routing Component
symfony/security             v4.1.6  v4.1.6  Symfony Security Component
symfony/security-acl         v3.0.1  v3.0.1  Symfony Security Component - ACL (Access Control List)
symfony/security-bundle      v4.1.6  v4.1.6  Symfony SecurityBundle
symfony/serializer           v4.1.6  v4.1.6  Symfony Serializer Component
symfony/stopwatch            v4.1.6  v4.1.6  Symfony Stopwatch Component
symfony/swiftmailer-bundle   v3.2.3  v3.2.3  Symfony SwiftmailerBundle
symfony/templating           v4.1.6  v4.1.6  Symfony Templating Component
symfony/translation          v4.1.6  v4.1.6  Symfony Translation Component
symfony/twig-bridge          v4.1.6  v4.1.6  Symfony Twig Bridge
symfony/twig-bundle          v4.1.6  v4.1.6  Symfony TwigBundle
symfony/validator            v4.1.6  v4.1.6  Symfony Validator Component
symfony/var-dumper           v4.1.6  v4.1.6  Symfony mechanism for exploring and dumping PHP variables
symfony/web-profiler-bundle  v4.1.6  v4.1.6  Symfony WebProfilerBundle
symfony/web-server-bundle    v4.1.6  v4.1.6  Symfony WebServerBundle
symfony/webpack-encore-pack  v1.0.3  v1.0.3  A pack for Symfony Encore
symfony/yaml                 v4.1.6  v4.1.6  Symfony Yaml Component

```

#### PHP version

```
> php -v
PHP 7.2.10-0ubuntu1 (cli) (built: Sep 13 2018 13:38:55) ( NTS )
Copyright (c) 1997-2018 The PHP Group
Zend Engine v3.2.0, Copyright (c) 1998-2018 Zend Technologies
    with Zend OPcache v7.2.10-0ubuntu1, Copyright (c) 1999-2018, by Zend Technologies
    with Xdebug v2.6.0, Copyright (c) 2002-2018, by Derick Rethans

```

## Subject

Wrong type hint it should be "array" instead of "string" added here:

https://github.com/sonata-project/SonataFormatterBundle/commit/aebfa25df39cfb6e3a980697c6f4223e24a73783#diff-d3a514a2fb8ee64646e7f570b8f00104R172

## Steps to reproduce

In a ckeditor: 
1. Click in the button "add image"
2. Click "Browse server"
3.  The error appears.

## Expected results

The browse page should work

## Actual results

Argument 2 passed to Sonata\FormatterBundle\Controller\CkeditorAdminController::setFormTheme() must be of the type string, array given, called in vendor/sonata-project/formatter-bundle/src/Controller/CkeditorAdminController.php on line 78

**Comment by kunicmarko20** [comment]
Yep,  you are right

https://github.com/sonata-project/SonataAdminBundle/blob/3.x/src/Admin/AbstractAdmin.php#L403

Can you submit a PR?

**Comment by jmontoyaa** [comment]
PR sent

**Comment by jordisala1991** [comment]
This should be fixed by #371 and already released on 4.1.2

**Comment by jordisala1991** [comment]
This should be fixed by #371 and already released on 4.1.2
